.papr.yml

```diff
@@ -112,6 +112,15 @@ tests:
     -v $(pwd):/srv/code -w /srv/code
     registry.fedoraproject.org/fedora:27 /bin/sh -c
     "./ci/build.sh && make install && ./tests/compose"
+  # And now one that isn't --privileged https://github.com/projectatomic/rpm-ostree/issues/1329
+  - docker run --rm --security-opt seccomp=unconfined
+    --security-opt label=type:spc_t \
```

Comment:
I think you mentioned this somewhere, though why do we need `spc_t`? Is it to let SELinux allows through the `unshare` call? Can you add a comment above about that?

Reply:
For the record - nope, it's because policy denies the `mount` syscall.

Reply:
I think the mount syscall is allowed, but it matters what is being mounted and on to what.